src/api/factories/CancelBidFactory.ts

```diff
@@ -0,0 +1,22 @@
+import { inject, named } from 'inversify';
```

Comment:
I dont see the need for CancelBidFactory, there's BidFactory which takes in BidMessage and should convert it to Bid